src/internal_events/process.rs

```diff
@@ -17,6 +18,15 @@ impl InternalEvent for VectorStarted {
             arch = built_info::TARGET_ARCH,
             build_id = built_info::VECTOR_BUILD_DESC.unwrap_or("none"),
         );
+        gauge!(
+            "vector_info",
```

Comment:
I think we could just call this `build_info` as it is already in the `vector` namespace. As Spencer shows, this currently ends up as `vector_vector_info` which is a bit redundant.

Reply:
😆 missed that completely

Reply:
Yup, was going to comment the same, that the metric should be `vector_build_info` as per common practices.
See https://www.robustperception.io/exposing-the-software-version-to-prometheus/